L1Trigger/Phase2L1ParticleFlow/src/firmware/data.h

```diff
@@ -0,0 +1,209 @@
+#ifndef FIRMWARE_DATA_H
```

Comment:
include guard should contain subsystem and package names

Reply:
Ok, updated all the header guards of the headers that were missing the subsystem and package (as they were coming from elsewhere)